agent/src/agent/scripts/migrate-to-db.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import json
+import logging
+import os
+import sys
+
+from agent import logger
+from agent import source, pipeline, destination
+from agent.constants import MONITORING_SOURCE_NAME
+
+logger_ = logger.get_logger('scripts.migrate-to-db.run')
+handler = logging.StreamHandler(sys.stdout)
+handler.setFormatter(logging.Formatter('%(asctime)s - %(levelname)s - %(message)s'))
+logger_.addHandler(handler)
+
+
+def run(data_dir):
+    migrate_destination(data_dir)
+    migrate_sources(data_dir)
+    migrate_pipelines(data_dir)
+
+
+def migrate_destination(data_dir):
+    with open(os.path.join(data_dir, 'destination.json')) as f:
+        if destination.repository.exists():
+            print('Destination already exists, skipping')
+            return
+        data = json.load(f)
+        dest = destination.HttpDestination()
+        dest.config = data['config']
+        dest.host_id = data['host_id']
+        dest.access_key = data['access_key']
+        destination.repository.create(dest)
+        print('Destination successfully migrated')
+
+
+def migrate_sources(data_dir):
+    if not source.repository.exists(MONITORING_SOURCE_NAME):
+        source.repository.create(source.Source(MONITORING_SOURCE_NAME, source.TYPE_MONITORING, {}))
+        print('Created monitoring source')
+    else:
+        print('Monitoring source already exists')
+    for root, _, filenames in os.walk(os.path.join(data_dir, 'sources')):
+        for filename in filenames:
```

Comment:
It's better to catch every exception in the loop, log it, and continue with the next file. Just in case something is broken in the user's env (which often is 😔). The same for pipelines